README.md

```diff
@@ -0,0 +1,118 @@
+This is a project to build a small cloud image that has useful tools and function for debugging or developing cloud infrastructure. Supported architectures are:
+
+ * aarch64 (64-bit Armv8 often called arm64)
+ * arm (32-bit Armv7)
+ * i386 (32-bit x86)
+ * ppc (32-bit PowerPC)
+ * ppc64 (64-bit Power big endian)
+ * ppc64le (64-bit Power little endian)
+ * x86_64 (64-bit x86 often called amd64)
+
+Resulting images can be booted using QEMU. Several ways of booting are provided:
+
+ * UEFI (aarch64, arm, i386, x86_64)
+ * BIOS (i386, x86_64)
+ * OpenFirmware (ppc, ppc64, ppc64le)
+ * direct kernel + initrd (all architectures)
+
+# Build instructions
+
+The following works on Ubuntu 18.04 LTS, running on x86_64.  Native building for other architectures is not recommended. Support for building under other distributions is limited and not tested.
+
+## Clone sources
+
+ * git clone https://github.com/cirros-dev/cirros.git
+   or
+ * git clone git@github.com:cirros-dev/cirros.git
+
+## Short version
+
+Once you fetched source code all you have to do something like:
+
+ * cd cirros
+ * bin/build-release daily
+
+If you want to build for only subset of supported architectures when use ARCHES variable like:
+
+* ARCHES=aarch64,x86_64,arm bin/build-release daily
+
+Resulting images will be present in ../build-dYYMMDD/release directory.
+
+
+## Long, detailed version
+
+To use it, you would do something like:
+
+ * cd cirros
+
+ * get the build dependencies:
+```bash
+   $ ./bin/system-setup
```

Comment:
I must not have hit 'save' on this...
my comment was just to move this to the 'short version'.

If you don't run this, you're just going to fail at some seconds or minutes later due to missing some dependency.